weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallOverview.tsx

```diff
@@ -23,19 +31,26 @@ export const CallName = styled.div<{$isEditing?: boolean}>`
   font-size: 16px;
   font-weight: 600;
   text-align: left;
-  word-break: break-all;
-  width: ${props => (props.$isEditing ? '100%' : 'max-content%')};
+  min-width: 0;
+  display: inline-block;
```

Comment:
If we don't need $isEditing we can get rid of it and there is a cascade of cleanup we can do

Reply:
I _think_ it's unnecessary